lib/debug/session.rb

```diff
@@ -231,25 +231,36 @@ def wait_command
       when 'c', 'continue'
         @tc << :continue
 
-      # * `q[uit]` or exit or `Ctrl-D`
+      # * `q[uit]` or `Ctrl-D`
       #   * Finish debugger (with the debuggee process on non-remote debugging).
-      when 'q', 'quit', 'exit'
+      when 'q', 'quit'
         if ask 'Really quit?'
           @ui.quit arg.to_i
           @tc << :continue
         else
           return :retry
         end
 
-      # * `kill` or `q[uit]!`
+      # * `q[uit]!`
+      #   * Finish debugger immediately (with the debuggee process on non-remote debugging).
+      when 'q!', 'quit!'
+        @ui.quit arg.to_i
+        @tc << :continue
+
+      # * `kill`
       #   * Stop the debuggee process.
```

Comment:
This is my fault, but could you change the description about `kill` command?
Maybe the word "immediately" fits for this command. For example,

"Stop the debugger process immediately with `exit!` method"

is clear for Rubyist (I think rubyists knows what `exit!` method mean. ... is it ok?)

So the doc for `quit!` should be:

```
* q[uit]!
  * Same as `quit` command, but without asking prompt. <- please correct my poor English please :p
```

and so on will be clear.

Reply:
maybe `q!` is familiar with vim's command for programmers.